http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -76,8 +78,53 @@ paths:
                 $ref: "#/components/schemas/LeonardoVersion"
       security: []
 
-  ## Runtime paths ##
+  ## All resources path ##
+  "/api/google/v1/resources/{googleProject}/deleteAll":
+    delete:
+      summary: Deletes ALL runtimes and apps associated with the given google project
+      description: deletes all runtimes and apps in a google project
+      operationId: deleteAllResources
+      tags:
+        - resources
+      parameters:
+        - in: path
+          name: googleProject
+          description: googleProject
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: deleteInCloud
+          description: Whether or not the runtimes and apps should be deleted in the cloud. If set to false, no call to the cloud will be made and the resources will be marked as deleted in SAM and Leonardo databases.
```

Comment:
Maybe worth specifying what the default is here? Based on reading the code it seems the default is `false`.